app/models/source.rb

```diff
@@ -11,6 +11,7 @@ class Source < ApplicationRecord
 
   attribute :availability_status, :string
   validates :availability_status, :inclusion => { :in => %w[available partially_available unavailable] }, :allow_nil => true
+  validates :app_creation_workflow, :inclusion => {:in => %w[manual-configuration account-authorization]}
```

Comment:
Oh, sorry I wrote it wrong, but we should keep it consistent with availibility_status and use `_` too (`manual_configuration` etc)